README.md

```diff
@@ -49,10 +53,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Report Deploy
-        uses: OpsLevel/report-deploy-github-action@v0.10.0
+        uses: OpsLevel/report-deploy-github-action@v0.11.0
         with:
           integration_url: ${{ secrets.DEPLOY_INTEGRATION_URL }}
           service: "my-service"
+          use_docker: "true"
```

Comment:
instead of having this be a flag should we just auto detect if the host system is linux and say - use docker - and then if not - use bash?

```
If you want to run workflows that use Docker container actions or service containers, you must use a Linux machine and Docker must be installed.
```

Reply:
Idk I feel like this is assuming a lot (even if it's right). I think most folks won't even need to use this flag but at least it puts the ball in their court